client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/index.tsx

```diff
@@ -62,66 +59,45 @@ const HostingContent = ( {
 	);
 };
 
-export default function HostingV2( { onAddToCart }: Props ) {
+export default function HostingV2( { onAddToCart, section }: Props ) {
 	const translate = useTranslate();
 
 	const isLargeScreen = useBreakpoint( '>1280px' );
 
-	const { setParams, getParamValue } = useURLQueryParams();
-
-	const [ selectedFeatureId, setSelectedFeatureId ] = useState< string >(
-		getParamValue( 'hosting' )
-	);
-
-	useLayoutEffect( () => {
```

Comment:
Since we have removed this, can we do something similar for the redirect? Ideally, when visiting `/hosting`, the user should be redirected to `/hosting/wpcom` since that is the default tab.

Reply:
Good point! made some adjust so it redirects to `/hosting/wpcom` by default 👍